// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/conformance/declarationEmit/leaveOptionalParameterAsWritten.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -7,6 +7,6 @@
 {"version":3,"file":"b.d.ts","sourceRoot":"","sources":["../b.ts"],"names":[],"mappings":"AAAA,OAAO,KAAK,CAAC,MAAM,KAAK,CAAC;AACzB,OAAO,CAAC,MAAM,CAAC;IACb,UAAU,KAAK,CAAC;QACd,UAAiB,OAAO,CAAC;YACvB,MAAM,QAAQ,GAAG,GAAG,CAAC,CAAC,GAAG,CAAC;SAC3B;KACF;CACF"}
 
 
 //// [/.src/dist/c.d.ts.map]
-{"version":3,"file":"c.d.ts","sourceRoot":"","sources":["../c.ts"],"names":[],"mappings":"AAAA,KAAK,GAAG,GAAG,KAAK,CAAC,OAAO,CAAC,GAAG,CAAC;AAC7B,eAAO,MAAM,GAAG,OAAQ,GAAG,KAAG,IAAU,CAAA"}
+{"version":3,"file":"c.d.ts","sourceRoot":"","sources":["../c.ts"],"names":[],"mappings":"AAAA,KAAK,GAAG,GAAG,KAAK,CAAC,OAAO,CAAC,GAAG,CAAC;AAC7B,eAAO,MAAM,GAAG,GAAI,CAAC,CAAC,EAAE,GAAG,KAAG,IAAU,CAAA"}
 
